app/src/utils/AppUtils.js

```diff
@@ -40,3 +41,13 @@ export const isAppOpenedInIframe = () => {
     return true;
   }
 };
+
+export const isAppTypeSessionBear = () => {
```

Comment:
rename to `isSessionBearApp` because, at least so far,  introducing new grammar of `AppType` does not seem necessary.